README.md

```diff
@@ -170,6 +170,16 @@ $allNonInitiatedOrPendingModels = Model::otherCurrentStatus(['initiated', 'pendi
 $allNonInitiatedOrPendingModels = Model::otherCurrentStatus('initiated', 'pending');
 ```
 
+### Retrieving models that have current status created between two dates
```

Comment:
What do you mean with "current status". Do you mean any status?

Reply:
I mean by current status latest status.